scripts/verifyMessages.ps1

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env pwsh
+
+# Validate input
+if ($args.Count -ne 3) {
+    Write-Host "Usage: <script> <directory-to-search> <cpp-messages.json> <artifact-messages.json>"
+    exit 1
+}
+
+$SEARCH_DIR = $args[0]
```

Comment:
Stupid question: Instead of having the caller tell us what to check, should this script use $PSScriptRoot to figure it  out?

Reply:
Not a stupid question. That makes sense to me.